lib/src/airship.dart

```diff
@@ -181,6 +181,20 @@ class Airship {
     return await _channel.invokeMethod('removeTags', tags);
   }
 
+  static Future<void> editChannelTagGroups(List<Map<String, dynamic>> operations) async {
```

Comment:
Can we make a model object to define the operations sorta like we do in react native?

Reply:
From what I can see, the operations are defined like this in react native
`var operation = { "operationType": "add", "group": group, "tags": tags }`
You mean create something like a tag group editor? https://github.com/urbanairship/react-native-module/blob/master/js/TagGroupEditor.js

Reply:
Yes, the tag group editor. That way the underlying operation payload is hidden from the user.